auditbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -4,6 +4,8 @@
 {beatname_uc} <<running-on-docker,Docker images>> can be used on Kubernetes to
 check files integrity.
 
+Running {ecloud} on Kubernetes? See {eck-ref}/k8s-beat.html[Run {beats} on ECK].
```

Comment:
Would there be a way to make it a bit more visible? My feeling is that it looks inconspicuous given that a whole product and a very different way of deploying Beats is behind it. WDYT?

Reply:
@david-kow What do you have in mind? We can make the text bold, but I don't think that's what you mean. :-) 

Do you mean a separate section that gives more info about running Beats on ECK--that maybe describes how it's different or easier?

I read through the cloud docs and there wasn't much there to guide me.

Reply:
Separate section (`Running on Kubernetes with ECK`?) would definitely tick the box for visibility, but I understand we might not have enough content to fill it up. Unless a paragraph of description and a link to docs on ECK side would suffice?

A tip? A note? A separate title? I would be fine with anything (current proposal included), I just want to make sure we explored all options.

Let me know if there is anything I can help with, I'd be glad to.